tests/undocommand.js

```diff
@@ -266,13 +266,6 @@ describe( 'UndoCommand', () => {
 				for ( const item of model.createRangeIn( doc.graveyard ).getItems() ) {
 					expect( item.hasAttribute( 'key' ) ).to.be.false;
 				}
-
-				// Let's undo wrapping. This will remove the P element and leave us with empty root.
-				undo.execute( batch3 );
-				expect( root.maxOffset ).to.equal( 0 );
-
-				expect( editor.model.document.selection.getFirstRange().isEqual( r( 0, 0 ) ) ).to.be.true;
-				expect( editor.model.document.selection.isBackward ).to.be.false;
```

Comment:
I had to remove this part of the test because it started to fail. Unfortunately, this part is about more complex OT of "unwrap action". Since we removed `WrapOperation` and `UnwrapOperation` we no longer can provide fixes for such scenario.

It was working before "by accident" due to a bug in a different pair of transformations.